plugin/src/PatchTree.lua

```diff
@@ -341,6 +356,8 @@ local function build(patch, instanceMap, changeListHeaders)
 	return tree
 end
 
+-- Updates the metadata of a tree with the unapplied patch and currently existing instances
+-- Builds a new tree from the data if one does not exist
 local function updateMetadata(tree, patch, instanceMap, unappliedPatch)
```

Comment:
Same thing here, if the function name was just `updateTreeMetadata` it's obvious just from the signature that it updates metatable of tree with the given args.

Reply:
Same response as above- PatchTree already says tree in it so I removed the repeated word